src/amo/pages/UserProfileEdit/index.js

```diff
@@ -568,7 +568,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     fields are optional, but they'll help other users get to
                     know %(username)s better.`,
                       ),
-                      { username },
+                      { username: user ? user.username : userId },
```

Comment:
Same comment about `userId` not being particularly useful here.

Reply:
Same answer 😛

Reply:
I changed to use a `LoadingText` component when there is no user:

![screen shot 2018-11-19 at 12 15 19](https://user-images.githubusercontent.com/217628/48703720-ee751c00-ebf4-11e8-8caf-06c9708c062c.png)